src/main/features/response/views/statement-of-means/explanation.njk

```diff
@@ -0,0 +1,18 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, textInput, saveAndContinueButton, errorSummary, textArea %}
```

Comment:
please remove unused imports

Reply:
Good spot, something that slipped last review. I removed all unused imports from this and other SoM views.